flink-staging/flink-table/pom.xml

```diff
@@ -59,6 +59,12 @@ under the License.
 		</dependency>
 
 		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-hcatalog</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+
+		<dependency>
```

Comment:
Is it necessary to have the hcatalog dependency here or could it be put into another package?